docs/ref/validators.txt

```diff
@@ -222,12 +222,15 @@ to, or in lieu of custom ``field.clean()`` methods.
 ``int_list_validator``
 ----------------------
 
-.. function:: int_list_validator(sep=',', message=None, code='invalid')
+.. function:: int_list_validator(sep=',', message=None, code='invalid', allow_negative=False)
 
     .. versionadded:: 1.9
 
+    .. versionchanged:: 1.10
+
     Returns a :class:`RegexValidator` instance that ensures a string
-    consists of integers separated by ``sep``.
+    consists of integers separated by ``sep``. It allows negative integers when
+    ``allow_negative`` is ``True``.
```

Comment:
I am a bit confused whether I should write the new feature and the existing class description 1) separately under the versionadded and versionchanged block or
2) the way I have done it right now.